### DISCUSSION TITLE
Teleport API access for other languages

### DISCUSSION THREAD

**Discussion by Souldiv** [gravitational/teleport#36502] [Q&A] [answered]
Is there any planned work for using other languages such as Python, Rust to have access to the to teleport API for creating clients? Is there an alternative for this?

**Comment by webvictim** [comment, marked as answer]
There are not currently any plans to do this. There is a tracking issue here: https://github.com/gravitational/teleport/issues/32658

**Reply by Souldiv** [reply]
got it. Is the teleport api supported to be built as a c shared library?

**Reply by webvictim** [reply]
No, it's all written in Go.
